appengine/flexible/WebSocketSignalR/wwwroot/js/chat.js

```diff
@@ -0,0 +1,38 @@
+"use strict";
```

Comment:
I think this file needs a copyright statement too.

Reply:
Added. PTAL :)